### ISSUE TITLE
Configuration to avoid opening the diagnostics console

### ISSUE THREAD

**Issue by EstebanBorai** [sublimelsp/LSP-rust-analyzer#45]
It would be nice to have an configuration option to avoid opening the console when a diagnostics warning comes up.

## Screenshot

![image](https://user-images.githubusercontent.com/34756077/141661444-6251af61-fe62-4578-b41a-4e22e1d850c9.png)

I'm testing the LSP for Rust Analyzer for the first time and I'm having a great time with it so I'm not sure if perhaps I'm missing something?

This is all the _diagnostics_ settings I have available as of now:

```json
		"rust-analyzer.debug.sourceFileMap": {"/rustc/<id>": "${env:USERPROFILE}/.rustup/toolchains/<toolchain-id>/lib/rustlib/src/rust"},
		//List of rust-analyzer diagnostics to disable.
		"rust-analyzer.diagnostics.disabled": [],
		//Whether to show native rust-analyzer diagnostics.
		"rust-analyzer.diagnostics.enable": true,
		//Whether to show experimental rust-analyzer diagnostics that might have more false positives than usual.
		"rust-analyzer.diagnostics.enableExperimental": true,
		//Map of prefixes to be substituted when parsing diagnostic file paths. This should be the reverse mapping of what is passed to `rustc` as `--remap-path-prefix`.
		"rust-analyzer.diagnostics.remapPrefix": {},
		//List of warnings that should be displayed with hint severity.  The warnings will be indicated by faded text or three dots in code and will not show up in the `Problems Panel`.
		"rust-analyzer.diagnostics.warningsAsHint": [],
		//List of warnings that should be displayed with info severity.  The warnings will be indicated by a blue squiggly underline in code and a blue icon in the `Problems Panel`.
		"rust-analyzer.diagnostics.warningsAsInfo": [],
		//Expand attribute macros.
```

**Comment by EstebanBorai** [comment]
Nevermind,  I got the desired behavior by disabling the  diagnostics panel on a top level, using the LSP Settings.

```json
"show_diagnostics_panel_on_save": 0,
```

Thanks anyways!
